src/components/tab-bar/tab.types.ts

```diff
@@ -0,0 +1,26 @@
+export interface Tab {
```

Comment:
It might be a good idea to also have an `id` property so consumers have a way of identifying their tabs